migration_task.go

```diff
@@ -0,0 +1,56 @@
+package migrate
+
+import "github.com/golang-migrate/migrate/v4/database"
+
+// MigrationTask is a callback function type that can be used to execute a
+// Golang based migration step after a SQL based migration step has been
+// executed. The callback function receives the migration and the database
```

Comment:
"after a SQL based migration step has been executed." is this comment now stale?